data/src/main/java/com/globant/data/service/CharacterService.kt

```diff
@@ -5,23 +5,30 @@ import com.globant.data.ZERO
 import com.globant.data.mapper.CharacterMapperService
 import com.globant.data.service.api.MarvelApi
 import com.globant.domain.entities.MarvelCharacter
+import com.globant.domain.services.MarvelCharactersService
 import com.globant.domain.utils.Result
 
-class CharacterService {
+class CharacterService : MarvelCharactersService {
```

Comment:
for a simple naming convention change the name of this class to MarvelCharactersServiceImpl instead of CharacterService

Reply:
Done